private/buf/bufformat/bufformat.go

```diff
@@ -66,3 +68,8 @@ func Format(ctx context.Context, module bufmodule.Module) (_ storage.ReadBucket,
 	}
 	return readWriteBucket, nil
 }
+
+func FormatFileNode(fileNode *ast.FileNode, dest io.Writer) error {
```

Comment:
No godoc, reverses order of arguments without an explanation, PR has no context as to why this is needed